src/cl/prysm/prysm_launcher.star

```diff
@@ -8,33 +8,52 @@ package_io = import_module("github.com/kurtosis-tech/eth-network-package/package
 
 IMAGE_SEPARATOR_DELIMITER	= ","
 EXPECTED_NUM_IMAGES			= 2
-
+#  ---------------------------------- Beacon client -------------------------------------
 CONSENSUS_DATA_DIRPATH_ON_SERVICE_CONTAINER			= "/consensus-data"
 GENESIS_DATA_MOUNT_DIRPATH_ON_SERVICE_CONTAINER		= "/genesis"
-VALIDATOR_KEYS_MOUNT_DIRPATH_ON_SERVICE_CONTAINER	= "/validator-keys"
-PRYSM_PASSWORD_MOUNT_DIRPATH_ON_SERVICE_CONTAINER	= "/prysm-password"
+
 
 # Port IDs
 TCP_DISCOVERY_PORT_ID		= "tcp-discovery"
 UDP_DISCOVERY_PORT_ID		= "udp-discovery"
 RPC_PORT_ID					= "rpc"
 HTTP_PORT_ID				= "http"
 BEACON_MONITORING_PORT_ID	= "monitoring"
-VALIDATOR_MONITORING_PORT_ID= "monitoring"
 
 # Port nums
 DISCOVERY_TCP_PORT_NUM		= 13000
 DISCOVERY_UDP_PORT_NUM		= 12000
 RPC_PORT_NUM				= 4000
 HTTP_PORT_NUM				= 3500
 BEACON_MONITORING_PORT_NUM	= 8080
+
+# The min/max CPU/memory that the beacon node can use
+BEACON_MIN_CPU = 50
+BEACON_MAX_CPU = 1000
+BEACON_MIN_MEMORY = 256
+BEACON_MAX_MEMORY = 1024
+
+#  ---------------------------------- Validator client -------------------------------------
+VALIDATING_REWARDS_ACCOUNT = "0x878705ba3f8Bc32FCf7F4CAa1A35E72AF65CF766"
```

Comment:
this is just fee recipient? what address is this set to? is it one of the genesis allocation addresses?

Reply:
Yes.